darts/models/exponential_smoothing.py

```diff
@@ -75,11 +76,15 @@ def fit(self, series: TimeSeries):
                                            trend=self.trend.value,
                                            damped_trend=self.damped,
                                            seasonal=self.seasonal.value,
-                                           seasonal_periods=self.seasonal_periods)
-
+                                           seasonal_periods= None if self.infer_seasonal_periods else self.seasonal_periods,
+                                           freq=series.freq,
```

Comment:
From the statsmodels documentation "this is optional if dates are given". Do we need it?